Python/Endpoint Examples/Multipart Payload/pdf-with-redacted-text-preview.py

```diff
@@ -0,0 +1,43 @@
+from requests_toolbelt import MultipartEncoder
+import requests
+import json
+
+pdf_with_redacted_text_endpoint_url = 'https://api.pdfrest.com/pdf-with-redacted-text-preview'
+
+redaction_options = [{
+        "type": "preset",
+        "value": "uuid",
```

Comment:
```suggestion
        "value": "email",
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```